### PR TITLE
chore: update steps to download binary to include both options

### PR DESCRIPTION
**Pull request by canhtrinh**

### CODE REVIEW

File: src/pages/validator/amplifier/integrator-onboarding.mdx [hunk 1]
```diff
@@ -14,7 +14,13 @@
...
 ## Set up your CosmWasm devnet environment
 
-1. Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device.
+1. Install the `v0.35.5` axelard binary. You can either:
+    * Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device. You will also need to download 
+    [`wasmvm v1.3.1`](https://github.com/CosmWasm/wasmvm/releases/tag/v1.3.1) `libwasmvm.x86_64.so` (for Linux) or `libwasmvm.dylib` (for Mac) and in either case add the parent folder to `LD_LIBRARY_PATH`, OR: 
+    * Build `wasm` enabled `axelard` binary directly from the `axelar-core`: 
+        * Clone the [repo](https://github.com/axelarnetwork/axelar-core/tree/cosmwasm)
+        * Checkout the v0.35.5 release tag
+        * Run `make WASM_ENABLED=true build`
```
Comment (milapsheth) [inline src/pages/validator/amplifier/integrator-onboarding.mdx:23]:
```suggestion
        * Run `make build`
```
it's done by default (devnet wasm doc was really old, need to use the normal build instructions)

File: src/pages/validator/amplifier/integrator-onboarding.mdx [hunk 2]
```diff
@@ -14,7 +14,13 @@
 
 ## Set up your CosmWasm devnet environment
 
-1. Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device.
+1. Install the `v0.35.5` axelard binary. You can either:
+    * Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device. You will also need to download 
+    [`wasmvm v1.3.1`](https://github.com/CosmWasm/wasmvm/releases/tag/v1.3.1) `libwasmvm.x86_64.so` (for Linux) or `libwasmvm.dylib` (for Mac) and in either case add the parent folder to `LD_LIBRARY_PATH`, OR: 
+    * Build `wasm` enabled `axelard` binary directly from the `axelar-core`: 
+        * Clone the [repo](https://github.com/axelarnetwork/axelar-core/tree/cosmwasm)
```
Comment (milapsheth) [inline src/pages/validator/amplifier/integrator-onboarding.mdx:21]: linking to an invalid branch 
```suggestion
        * Clone the [repo](https://github.com/axelarnetwork/axelar-core)
```

File: src/pages/validator/amplifier/integrator-onboarding.mdx [hunk 3]
```diff
@@ -14,7 +14,13 @@
 
 ## Set up your CosmWasm devnet environment
 
-1. Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device.
+1. Install the `v0.35.5` axelard binary. You can either:
+    * Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device. You will also need to download 
+    [`wasmvm v1.3.1`](https://github.com/CosmWasm/wasmvm/releases/tag/v1.3.1) `libwasmvm.x86_64.so` (for Linux) or `libwasmvm.dylib` (for Mac) and in either case add the parent folder to `LD_LIBRARY_PATH`, OR: 
+    * Build `wasm` enabled `axelard` binary directly from the `axelar-core`:
```
Comment (milapsheth) [inline src/pages/validator/amplifier/integrator-onboarding.mdx:20]:
```suggestion
    * Build `axelard` binary directly from the `axelar-core`: 
```

File: src/pages/validator/amplifier/integrator-onboarding.mdx [hunk 4]
```diff
@@ -14,7 +14,13 @@
 
 ## Set up your CosmWasm devnet environment
 
-1. Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device.
+1. Install the `v0.35.5` axelard binary. You can either:
+    * Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device. You will also need to download 
+    [`wasmvm v1.3.1`](https://github.com/CosmWasm/wasmvm/releases/tag/v1.3.1) `libwasmvm.x86_64.so` (for Linux) or `libwasmvm.dylib` (for Mac) and in either case add the parent folder to `LD_LIBRARY_PATH`, OR: 
+    * Build `wasm` enabled `axelard` binary directly from the `axelar-core`: 
+        * Clone the [repo](https://github.com/axelarnetwork/axelar-core/tree/cosmwasm)
+        * Checkout the v0.35.5 release tag
```
Comment (milapsheth) [inline src/pages/validator/amplifier/integrator-onboarding.mdx:22]:
```suggestion
        * Checkout the `v0.35.5` release tag
```

**Comment by canhtrinh** [comment]
closing this PR. @ffe9f8 committed the changes directly: https://github.com/axelarnetwork/axelar-docs/pull/832/commits/62f03951dc5b8d9de503aa9ab00b81a4f0df6445

**Pull request closed without merging**